secrets-sample.json

```diff
@@ -1,11 +1,11 @@
 {
   "database": {
-    "url": "<your db url here>"
+    "url": "mongodb://DeViLINSIDE:bnb2k18@ds233061.mlab.com:33061/bnb-18"
   },
   "facebook_auth": {
-    "client_ID": "<your client id here>",
-    "client_secret": "<your client secret here>",
-    "callbackURL": "<some URL here>",
+    "client_ID": "856799954512226",
+    "client_secret": "06c317af76e63988725ec896ba648fcd",
+    "callbackURL": "http://localhost:3000/auth/facebook/callback",
     "profile_fields": "['id', 'displayName', 'name', 'gender' , 'email', 'photos']"
   }
```

Comment:
Immediately revert all of this and squash your commits

Reply:
ya will do it